### PR TITLE
Bonic tasks as a tqble

### PR DESCRIPTION
**Pull request by talregev**
Add a boinc tasks as a table.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -65,10 +66,12 @@ type ActiveTask struct {
 	ElapsedTime       float64  `xml:"elapsed_time"`
 	WorkingSetSize    float64  `xml:"working_set_size"`
 	ProgressRate      float64  `xml:"progress_rate"`
+	FrictionDone      float64  `xml:"fraction_done"`
```
Comment (CallMeFoxie) [inline main.go]: Friction vs fraction? :)

File: main.go [hunk 2]
```diff
@@ -283,7 +335,16 @@ func buildString(clientStateReply *ClientStateReply) string {
...
-
+	sb.WriteString("\n")
+	sb.WriteString("# TYPE boinc_client_tasks gauge\n")
+	projects := clientStateReply.ClientState.Projects
+	wu := clientStateReply.ClientState.WorkUnits
+	apps := clientStateReply.ClientState.Apps
+	appvers := clientStateReply.ClientState.AppVersions
+	results := clientStateReply.ClientState.Results
+	for _, task := range results {
+		sb.WriteString(fmt.Sprintf("boinc_client_tasks{project=\"%s\",name=\"%s\",state=\"%d\",use=\"%s\",host=\"%s\",app=\"%s\"} %f\n", findProjectName(task.ProjectUrl, projects), task.Name, task.Activetask.State, task.Resources, host, findAppName(task.WUName, wu, appvers, apps), task.Activetask.FrictionDone*100))
```
Comment (CallMeFoxie) [inline main.go]: friction vs fraction again :)
also maybe `boinc_client_task_done_percent` (if it goes 0 - 1 or 0-100) would be better metric name?

File: main.go (hunk 1 above)
Comment (talregev) [reply main.go]: yes, my bad :) 
I will fix it. Thank you!

File: main.go (hunk 2 above)
Comment (talregev) [reply main.go]: yes, I can change that.
I also use this for the table. so it also for only tasks.

**Comment by talregev** [comment]
I am closing this one and open a new one :)

**Pull request closed without merging**